### ISSUE TITLE
Can't install with pip

### ISSUE THREAD

**Issue by KPLauritzen** [charnley/rmsd#19]
I started a new python2 virtualenv with only numpy installed.

I cloned this repository and changed to the `pip` branch. 
Trying to install I get the following error
```
(test_rmsd_2) primdal at Kaspers-MBP in ~/dev/rmsd at [13:58] (pip)
$ pip install .
Processing /Users/primdal/dev/rmsd
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/private/var/folders/6l/r_g4km7552zd0s6jyvlyn6100000gn/T/pip-zDQPdV-build/setup.py", line 11, in <module>
        from rmsd.calculate_rmsd import __version__
      File "rmsd/__init__.py", line 2, in <module>
        from rmsd.calculate_rmsd import *
      File "rmsd/calculate_rmsd.py", line 17, in <module>
        from builtins import range
    ImportError: No module named builtins

    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /private/var/folders/6l/r_g4km7552zd0s6jyvlyn6100000gn/T/pip-zDQPdV-build/
```

`builtins` seems to be a python3-only package

Doing the same with a python3 virtualenv is fine. No errors encountered

**Comment by larsbratholm** [comment]
passes tests with python2 now. Does it work with python3?

**Comment by charnley** [comment]
Fixed in c5c7bd5de3db34ef2a5996f0b64dd37c72e1c298
